### ISSUE TITLE
[Bug] Pasting multiple public keys breaks the SSH keys list

### ISSUE THREAD

**Issue by LucaRed** [hestiacp/hestiacp#4402]
### Describe the bug

I have a text file with some public keys that I always add to new users that I create. Example:
```
ssh-rsa AAAAB3NzaC1yc2E... email-address1@example.com
ssh-rsa AAAAB3NzaC1yc2E... email-address2@example.com
ssh-rsa AAAAB3NzaC1yc2E... email-address3@example.com
```

If I add every key one by one, everything works correctly.

If I paste the contents of the whole file into the "Add SSH keys" screen, I can save without validation errors but then the SSH keys list page breaks (the list looks empty).

Instead, if I paste the contents of my file directly into the `authorized_keys` file (e.g. via SSH), everything works and the SSH keys index page is correctly populated.

### Tell us how to replicate the bug

1. Add a new user.
2. In the Edit User screen, click "Manage SSH Keys".
3. Click "Add SSH Key".
4. In the SSH Key textarea, paste multiple SSH keys.
5. Save.
6. The index page (`/list/key`) now looks empty.

(The `authorized_keys` file has the pasted SSH keys).

### Which components are affected by this bug?

Control Panel Web Interface

### Hestia Control Panel Version

1.8.11

### Operating system

Ubuntu 22.04

**Comment by jaapmarcus** [comment]
It doesn't support multiple keys at once yet

**Comment by LucaRed** [comment]
Yes, but maybe the validation should reject multiple keys, instead of silently failing and breaking the list.
